lib/wsm-panel.js

```diff
@@ -228,6 +228,8 @@ const WsmPanel = Class(
   // Socket.IO Parser
 
   decodeSocketIoPacket: function(data) {
+    if (!prefs.enableSocketIo) return;
```

Comment:
Please use brackets and new lines like as follows so, the code style is consistent:
```
if (<expression>) {
  return;
}
```

(we should integrate eslint at some point)

Honza